libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/routeprogress/RouteLegProgressTest.java

```diff
@@ -17,77 +13,41 @@
 import static junit.framework.Assert.assertNull;
 import static junit.framework.Assert.assertTrue;
 
-import java.io.IOException;
-
 public class RouteLegProgressTest extends BaseTest {
 
-  // Fixtures
-  private static final String DIRECTIONS_PRECISION_6 = "directions_v5_precision_6.json";
-
   private DirectionsRoute route;
+  private RouteProgress routeProgress;
   private RouteLeg firstLeg;
 
   @Before
-  public void setup() throws IOException {
-    Gson gson = new GsonBuilder()
-      .registerTypeAdapterFactory(DirectionsAdapterFactory.create()).create();
-    String body = loadJsonFixture(DIRECTIONS_PRECISION_6);
-    DirectionsResponse response = gson.fromJson(body, DirectionsResponse.class);
-    route = response.routes().get(0);
+  public void setup() throws Exception {
```

Comment:
Same with `RouteLegProgressTest`.

Reply:
The same thing happens with `RouteProgressTest`, `RouteStepProgressTest`, `SnapToRouteTest` and `ToleranceUtilsTest`.